Modules/@babylonjs/react-native/android/src/main/java/com/babylonreactnative/EngineView.java

```diff
@@ -89,6 +114,35 @@ public void surfaceDestroyed(SurfaceHolder surfaceHolder) {
         this.renderRunnable = null;
     }
 
+    @Override
+    public void onSurfaceTextureAvailable(@NonNull SurfaceTexture surfaceTexture, int i, int i1) {
+        this.startRenderLoop();
+        transparentSurface = new Surface(surfaceTexture);
```

Comment:
`this.transparentSurface` instead of just `transparentSurface` for all the references to this member variable.